Mathlib/Logic/Equiv/Set.lean

```diff
@@ -62,18 +62,18 @@ theorem _root_.Set.preimage_equiv_eq_image_symm {α β} (S : Set α) (f : β ≃
 
 -- Porting note: increased priority so this fires before `image_subset_iff`
 @[simp high]
-protected theorem subset_image {α β} (e : α ≃ β) (s : Set α) (t : Set β) :
+protected theorem symm_image_subset_iff {α β} (e : α ≃ β) (s : Set α) (t : Set β) :
     e.symm '' t ⊆ s ↔ t ⊆ e '' s := by rw [image_subset_iff, e.image_eq_preimage]
-#align equiv.subset_image Equiv.subset_image
+#align equiv.subset_image Equiv.symm_image_subset_iff
 
```

Comment:
```suggestion
#align equiv.subset_image Equiv.symm_image_subset_iff

@[deprecated] alias subset_image := symm_image_subset_iff -- deprecated since 2024-01-19

```

Reply:
@j-loreaux Right, done.